defs.bzl

```diff
@@ -23,4 +23,5 @@ load("//internal:node.bzl",
 load("//internal:jasmine_node_test.bzl", "jasmine_node_test")
 load("//internal:node_install.bzl", "node_repositories")
 load("//internal:npm_install.bzl", "npm_install")
+load("//internal:yarn_install.bzl", "yarn_install")
```

Comment:
do users need to know or care which package manager we use?
1. maybe we should just keep the name 'npm' even though we don't run the npm tool. After all, we are installing NPM dependencies from the NPM registry.
2. other ideas for a name?

Reply:
Answering my own question - since the user has to give us a lock file, they do have to know which is used... I guess that's okay then.